mlflow/transformers/__init__.py

```diff
@@ -768,6 +792,7 @@ def log_model(
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
     :param example_no_conversion: {{ example_no_conversion }}
+    :param prompt_template: {{ prompt_template }}
```

Comment:
I think it's ok to directly write the description here because other flavors don't have this argument.

Reply:
i was thinking of having a template since otherwise i'd have to copy-paste the text into `log_model` as well

Reply:
makes sense :)